api/envoy/config/endpoint/v3/load_report.proto

```diff
@@ -159,6 +160,14 @@ message ClusterStats {
   // in the DropOverload policy.
   repeated DroppedRequests dropped_requests = 5;
 
+  // the request latencies in this cluster. Only HTTP requests are currently
+  // supported. This field is not aggregated over the load_reporting_interval
+  // due to the way Envoy histograms are flushed, so these will only
+  // have data for the configured metrics interval.
+  // The set of quantiles are the stringified values from
+  // HistogramStatisticsImpl::supportedQuantiles()
+  map<string, google.protobuf.DoubleValue> request_latency_percentiles = 7;
```

Comment:
I'd suggest making this a repeated, I don't think there is a big advantage in using `map` and you can then just make the quantiles doubles.